crates/cxx-qt-lib/src/qml/qqmlengine.rs

```diff
@@ -3,8 +3,26 @@
 //
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
-#[cxx::bridge]
+#[cxx_qt::bridge(cxx_file_stem = "qqmlengine")]
 mod ffi {
+    unsafe extern "C++Qt" {
+        include!("cxx-qt-lib/qqmlengine.h");
+        type QQmlEngine;
+
+        /// This signal is emitted when the QML loaded by the engine would like to exit from the event loop with the specified return code retCode.
+        #[qsignal]
```

Comment:
this requires `cxx_qt_lib::ConnectionType` but we are `cxx_qt_lib` so it then fails.

This will work after moving anything that `cxx-qt-gen` uses to `cxx-qt` types

Reply:
#843 moves those types into `cxx-qt` :tada: